### PR TITLE
Add link for project review best practices doc in project beta mailer

### PR DESCRIPTION
**Pull request by lcjohnso**
Add link for "Project Review Best Practices" doc in project mailer (reply for beta requests, project launch updates).

### CODE REVIEW

File: app/views/project_request_mailer/project_request.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <p>Thanks for requesting your project, <a href="<%= @url %>"><%= @name %></a>, be reviewed for <%= @type %>.</p>
 
-<p>Someone on our team will review your project and be in touch with you in the next few weeks about the next steps.</p>
+<p>Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. To learn more about the project review process, timeline, and best practices, please visit <a href="https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68/edit?usp=sharing">this link</a>.</p>
```
Comment (camallen) [inline app/views/project_request_mailer/project_request.html.erb:5]:
```suggestion
<p>Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. Read our <a href="https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68">project review process, timeline, and best practices document</a> to learn more.</p>
```

File: app/views/project_request_mailer/project_request.text.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@ Hi <%= @owner %>
 
 Thanks for requesting your project, <%= @name %>: <%= @url %>, be reviewed for <%= @type %>.
 
-Someone on our team will review your project and be in touch with you in the next few weeks about the next steps.
+Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. To learn more about the project review process, timeline, and best practices, please visit <%= link_to "this link", "https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68/edit?usp=sharing" %>.
```
Comment (camallen) [inline app/views/project_request_mailer/project_request.text.erb:5]: I'm not sure you should use the link_to helpers here as it generates html, ttps://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

You already have the full URL and this is a text template so just use the text :)
```suggestion
Someone on our team will review your project and be in touch with you in the next few weeks about the next steps. Read our project review process, timeline, and best practices document to learn more https://docs.google.com/document/d/1SJmOdGmpzYGyKpSnFt_tEe_BZIc2Bzmc3kKlWw-gX68.
```

**Review by camallen — APPROVED** [review]
LGTM - I think you should remove the link_to view helper as you already have the URL.

Some minor change suggestions on the text. 👍

File: app/views/project_request_mailer/project_request.text.erb (hunk 2 above)
Comment (lcjohnso) [reply app/views/project_request_mailer/project_request.text.erb:5]: Ah -- I now understand the `.text.erb` template a bit better.  None of the variables will produce clickable links in the body text (but providers like Gmail will typically convert these anyway), so just spell out the full URL.  Got it.  Thanks!

**Pull request merged**